tests/suites/main_test.function

```diff
@@ -281,6 +281,14 @@ int main(int argc, const char *argv[])
 #if defined(MBEDTLS_MEMORY_BUFFER_ALLOC_C) && \
     !defined(TEST_SUITE_MEMORY_BUFFER_ALLOC)
     unsigned char alloc_buf[1000000];
+#endif 
+    if( platform_setup() != 0 )
```

Comment:
Please put a comment here guiding developers that ```platform_setup``` should be called in the beginning.

Reply:
Done.